package.json

```diff
@@ -16,6 +16,7 @@
     "lerna-publish": "lerna publish from-package --no-push --force-publish --no-verify-access --no-verify-registry"
   },
   "devDependencies": {
+    "eslint": "^7.9.0",
```

Comment:
Let's add a `lint` script to the `scripts` object here so we can actually invoke this change! 😄

Reply:
I forgot to push it up, was on my box. It is now fixed!